bc_obps/registration/tests/test_middleware.py

```diff
@@ -0,0 +1,25 @@
+from django.http import HttpRequest
```

Comment:
You might want to ask a dev ops person for comments on this one; I don't have anything helpful to add

Reply:
This test was already checked by @dleard in this card: https://app.zenhub.com/workspaces/team-moose-sprint-board-60d216175a2689000ec774d0/issues/gh/bcgov/cas-registration/156